packages/aws-cdk/test/api/hotswap/lambda-functions-inline-hotswap-deployments.test.ts

```diff
@@ -0,0 +1,110 @@
+import { Lambda } from 'aws-sdk';
+import * as setup from './hotswap-test-setup';
+import { zipString } from '../../../lib/api/hotswap/lambda-functions';
+
+let mockUpdateLambdaCode: (params: Lambda.Types.UpdateFunctionCodeRequest) => Lambda.Types.FunctionConfiguration;
+let mockTagResource: (params: Lambda.Types.TagResourceRequest) => {};
+let mockUntagResource: (params: Lambda.Types.UntagResourceRequest) => {};
+let hotswapMockSdkProvider: setup.HotswapMockSdkProvider;
+
+beforeEach(() => {
+  hotswapMockSdkProvider = setup.setupHotswapTests();
+  mockUpdateLambdaCode = jest.fn();
+  mockTagResource = jest.fn();
+  mockUntagResource = jest.fn();
+  hotswapMockSdkProvider.stubLambda({
+    updateFunctionCode: mockUpdateLambdaCode,
+    tagResource: mockTagResource,
+    untagResource: mockUntagResource,
+  });
+});
```

Comment:
this entire file should be part of `lambda-functions-hotswap-deployments.test.ts`, no? @skinny85, thoughts?

Reply:
Nope! I actually specifically asked to move these tests out (this is in order to keep our test files small).